docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.rst

```diff
@@ -78,11 +74,20 @@ instead of recompiling it.
 For more details about OpenVINO model caching, see the
 :doc:`Model Caching Overview <../optimize-inference/optimizing-latency/model-caching-overview>`.
 
+| **Supported Inference Data Types**
+| The NPU plugin supports the following data types as inference precision of internal primitives:
+|    Floating-point data types: F32, F16
+|    Quantized data types: U8 (quantized models may be INT8 or mixed FP16-INT8)
+|    Computation precision for the HW is FP16.
+|
+| For more details on how to get a quantized model, refer to the
+  :doc:`Model Optimization guide <../../model-optimization>` and
+  :doc:`NNCF tool quantization guide <../../model-optimization-guide/quantizing-models-post-training/basic-quantization-flow>`.
 
 Supported Features and properties
 #######################################
 
-The NPU device is currently supported by AUTO and MULTI inference modes
+The NPU device is currently supported by AUTO and MULTI plugins.
```

Comment:
I would keep "inference modes" as some time ago we were asked to refrain from using the word 'plugin' - that's why we often say inference devices and inference modes.